CHANGELOG.rst

```diff
@@ -1,6 +1,21 @@
 Changelog
 =========
 
+0.8.0 (2024-08-22)
+------------------
+
+**New features**
+
+* Add Django CSP with configurable settings
+* Add SECURE_HSTS_SECONDS and CSRF_COOKIE_HTTPONLY settings
```

Comment:
Should we put a warning here for `SECURE_HSTS_SECONDS`?

I no sure that anyone will read it but it might be a reminder to put it in the other projects changelogs.

Reply:
What warning should we add? `SECURE_HSTS_SECONDS` is not configurable right?

Reply:
Not configurable, just warning that it is now set because it can apparently break websites.

Reply:
@alextreme do you have ideas for this? https://docs.djangoproject.com/en/5.1/ref/middleware/#http-strict-transport-security states that this can break the application if not all pages are served over https, but what should we write in the changelogs to prevent this from happening? Just a warning to indicate that everything must be served over https?

Reply:
@stevenbal afaik HSTS only applies if IS_HTTPS is set to True (the default), and if you disable IS_HTTPS that usage via HTTP is allowed. A warning to state that mixed HTTPS+HTTP installations seems sufficient in my eyes